build/nogo_config.json

```diff
@@ -305,7 +305,7 @@
   "misspell": {
     "exclude_files": {
       "/cgo/": "ignore cgo code",
-      "/external/": "no need to vet third party code",
+      "external/": "no need to vet third party code",
```

Comment:
why change it?

Reply:
misspell will find the misspell problem in the ```skywalking-eyes```. We need to skip it. the old config has a problem.

Reply:
fix all the lint config.